### PR TITLE
Delete files not necessary in the consumer plugin

### PR DESCRIPTION
**Pull request by Sidsector9**
### Description of the Change

Once this package is installed, it removes `.github`, tooling (*.json) and *.md files so that the consumer plugin doesn't  include them in the final release.

### Changelog Entry

> Added - Functionality to delete files that should not be part of plugin release

### Credits

Props @Sidsector9 

### Checklist:

- [ ] I agree to follow this project's [**Code of Conduct**](https://github.com/10up/.github/blob/trunk/CODE_OF_CONDUCT.md).
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [ ] All new and existing tests pass.

### CODE REVIEW

File: replace-namespace.sh [hunk 1]
```diff
@@ -19,3 +18,5 @@ find "$SCRIPT_DIR" -type f \( -name "*.php" -o -name "*.json" \) ! -name "$SCRIP
 	# Use perl for the replacement in each file
 	perl -pi -e "s/WP_Compat_Validation_Tool/$NEW_NAMESPACE/g" "$file"
 done
+
+cd 10up-lib/wp-compat-validation-tool && rm -rf .github .gitignore composer.json composer.lock CHANGELOG.md CONTRIBUTING.md README.md LICENSE.md CODE_OF_CONDUCT.md CREDITS.md
```
Comment (ravinderk) [inline replace-namespace.sh:22]: @Sidsector9, we should create a `.distignore` file instead of adding file names in the command.

Comment (Sidsector9) [reply replace-namespace.sh:22]: @ravinderk any reason for the suggestion? As I understand, a `.distignore` file is used at a distribution level, which the consumer plugin of this package is responsible for. This package is not distributed anywhere and the command you pointed is cleaning up its own files.

Maybe in the next iterations we can publish ZIP with releases which can be pulled by the consumers instead. IMO we can leave it as it is for now. Although, I am also tempted to try out Composer's [archive.exclude](https://getcomposer.org/doc/04-schema.md#archive) property.

Comment (faisal-alvi) [inline replace-namespace.sh:22]: @Sidsector9 should we use `*.md` and `*.lock` to cover all files instead of hard coding each file name?

**Review by faisal-alvi — APPROVED** [review]
LGTM, just a question asked below (not a blocker)

File: replace-namespace.sh (hunk 1 above)
Comment (Sidsector9) [reply replace-namespace.sh:22]: Yeah, I was being very careful with each file 😅 
I'll handle this in the next PR.

**Pull request merged**